src/runtime_src/core/edge/drm/zocl/zocl_kds.c

```diff
@@ -466,6 +720,12 @@ int zocl_command_ioctl(struct drm_zocl_dev *zdev, void *data,
 		goto out1;
 	}
 
+	/* Check whether client has already Open Context for this Command */
+	if (check_for_open_context(zdev, client, xcmd) < 0) {
```

Comment:
naming?  What does the function actually do, it checks but the check could be either good or bad?